src/main/java/org/ja13/eau/sixnode/electriccable/ElectricCable.kt

```diff
@@ -192,20 +202,39 @@ class ElectricCableRender(tileEntity: org.ja13.eau.node.six.SixNodeEntity, side:
     }
 }
 
+
+
 class PlayerHarmer(val electricalLoad: org.ja13.eau.sim.ElectricalLoad, private val insulationVoltage: Double, val location: Coordonate): org.ja13.eau.sim.IProcess {
 
     private fun harmFunction(distance: Double) = 1.0 - ( distance / 3.0)
 
     override fun process(time: Double) {
         val harmLevel = Math.max(0.0, (electricalLoad.u - 50 - insulationVoltage) / 500.0)
         val objects = location.world().getEntitiesWithinAABB(Entity::class.java, location.getAxisAlignedBB(4))
+
         for(obj in objects) {
+
```

Comment:
I see we are in the business of introducing random white space.

Reply:
You don't want to know what used to be in that white space.